core/build.gradle.kts

```diff
@@ -0,0 +1,114 @@
+import com.saveourtool.processbuilder.configureDetekt
+import com.saveourtool.processbuilder.configureDiktat
+
+plugins {
+    kotlin("multiplatform")
+}
+
+kotlin {
+    jvm {
+        jvmToolchain(11)
+        withJava()
+        testRuns["test"].executionTask.configure {
+            useJUnitPlatform()
+        }
+    }
+    js(IR) {
+        browser {
+            commonWebpackConfig {
+                cssSupport {
+                    enabled.set(true)
+                }
+            }
+        }
+    }
+
+    linuxX64()
+    macosX64()
+    mingwX64()
+
+    sourceSets {
+        all {
+            languageSettings.optIn("kotlin.RequiresOptIn")
+        }
+
+        val commonMain by getting {
+            dependencies {
+                api(libs.okio)
+
+                implementation(libs.kotlin.logging)
+                implementation(libs.kotlinx.datetime)
+                implementation(libs.kotlinx.coroutines.core)
+            }
+        }
+
+        val commonTest by getting
+
+        /**
+         * js is supported in order to allow using ProcessBuilder in common section
+         * js tests make no sense so common tests are targeted as commonNonJsTest
+         * commonNonJsMain is required in order to make commonNonJsTest resolvable
+         */
+        val commonNonJsMain by creating {
+            dependsOn(commonMain)
+        }
+
+        val commonNonJsTest by creating {
+            dependsOn(commonTest)
+            dependencies {
+                implementation(libs.okio.fakefilesystem)
+                implementation(libs.kotest.assertions.core)
+                implementation(kotlin("test-common"))
+                implementation(kotlin("test-annotations-common"))
+            }
+        }
+
+        val jvmMain by getting {
+            dependsOn(commonNonJsMain)
+            dependencies {
+                implementation(libs.slf4j)
```

Comment:
do you need `slf4j`? we use mu logging here

Reply:
https://github.com/oshai/kotlin-logging#version-4x-vs-previous-versions

Reply:
I get it as `latest consumer must provide SLF4J`. ProcessBuilder is a library, not a final artifact. So is it possible to use here only kotlin-logging and provide SLF4J only in save-cli

Reply:
Oh, ok
Should be added to tests only then